settings.gradle

```diff
@@ -61,6 +61,7 @@ includeWithFlags ':it:spring:boot2-mixed',          'java', 'relocate'
 includeWithFlags ':it:spring:boot2-mixed-tomcat9',  'java', 'relocate'
 includeWithFlags ':it:spring:boot2-tomcat8',        'java', 'relocate'
 includeWithFlags ':it:spring:boot2-tomcat9',        'java', 'relocate'
+includeWithFlags ':it:thrift',                      'java', 'publish', 'relocate'
```

Comment:
`:it:thrift:` does not fully explain what features are covered by it.
How about `:it:thrift-full-camel` or `:it:thrift:full-camel`?

Reply:
Also, it should be just `'java', 'relocate'`

Reply:
Ok too, I choose this module name for similar thrift problem maybe encounter in the future.